installscripts/jenkinscli/jobs/job_build_pack_api.sh

```diff
@@ -28,6 +28,11 @@ cat <<EOF | $JENKINS_CLI_CMD create-job "build-pack-api"
           <description></description>
           <defaultValue>master</defaultValue>
         </hudson.model.StringParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>request_id</name>
+          <description></description>
+          <defaultValue>none</defaultValue>
```

Comment:
are you sure "none"

Reply:
yes. null or empty value was creating issue while initiating the build.